README.md

```diff
@@ -33,12 +33,46 @@ Inspired/compatible with [react-native-sqlite-storage](https://github.com/andpor
 
 ```typescript
 interface QueryResult {
-  status: 0 | 1; // 0 for correct execution
-  message: string; // if status === 1, here you will find error description
-  rows: any[];
+  status?: 0 | 1; // 0 for correct execution
   insertId?: number;
+  rowsAffected: number;
+  message?: string;
+  rows?: {
+    /** Raw array with all dataset */
+    _array: any[];
+    /** The lengh of the dataset */
+    length: number;
+    /** A convenience function to acess the index based the row object
+     * @param idx the row index
+     * @returns the row structure identified by column names
+     */
+    item: (idx: number) => any;
```

Comment:
The item function is not available on the C++ responses I think, it is only injected from the javascript versions for TypeORM

Reply:
You are right. Its not part of C++ response, only a conveniency method, enhanced from JavaScript side. Removed from readme.